api/utils/aws/region.go

```diff
@@ -28,6 +32,76 @@ func IsUSGovRegion(region string) bool {
 	return strings.HasPrefix(region, USGovRegionPrefix)
 }
 
+// ShortRegionToRegion converts short region codes to regular region names. For
+// example, a short region "use1" maps to region "us-east-1".
```

Comment:
Is this mapping documented somewhere? Last time when I was looking for it I wasn't able to find it.

Reply:
Personal I think that it would be very helpful to add the link into comment.  In worst case the git link will be just unavailable.